cmd/clusterctl/cmd/delete.go

```diff
@@ -92,6 +94,11 @@ func init() {
 	deleteCmd.Flags().StringVar(&dd.kubeconfigContext, "kubeconfig-context", "",
 		"Context to be used within the kubeconfig file. If empty, current context will be used.")
 
+	deleteCmd.Flags().Float32Var(&dd.restQPS, "rest-qps", 0,
+		"Maximum QPS from the REST client to Kubernetes.")
+	deleteCmd.Flags().IntVar(&dd.restBurst, "rest-burst", 0,
```

Comment:
I named it this way because strictly speaking, I don't think these parameters can be defined in a kubeconfig file: https://kubernetes.io/docs/reference/config-api/kubeconfig.v1/

Reply:
that's true. just trying to find something that is easier for users who do not really know what a rest client is and how it fits in Kubernetes machinery

Reply:
Yeah either of those sounds good, I'll make that change.